ding/entry/main_league.py

```diff
@@ -0,0 +1,291 @@
+"""
+BUG: Stop in 2 iters
```

Comment:
What is this?

Reply:
@zxzzz0 This is a demo for MARL, which is not yet completed. Forget about it.